src/panels/lovelace/entity-rows/types.ts

```diff
@@ -7,6 +7,7 @@ export interface EntityConfig {
   name?: string;
   icon?: string;
   image?: string;
+  color?: string;
```

Comment:
Unrelated change?

Reply:
User was able to define own colors via yml configuration. 
Removed it, to keep focus on the feature itself.